src/js/components/Loader.js

```diff
@@ -52,7 +52,9 @@ class Loader extends React.Component {
   render() {
     const { className, innerClassName, flip, size, type } = this.props;
     const config = typeMap[type] || typeMap.ballScale;
-    const classes = classNames("loader horizontal-center", className);
+    const classes = this.props.supressHorizontalCenter
```

Comment:
Also just a #nit language thing but let's add another "p" to suppress > `suppressHorizontalCenter`